### ISSUE TITLE
Picking up nested def.* forms

### ISSUE THREAD

**Issue by savagematt** [cursiveclojure/cursive#1021]
I would love Cursive to be able to pick up the `defrecord` in this case:

![image](https://cloud.githubusercontent.com/assets/187214/9350827/d63dd89a-464b-11e5-9307-92bc7b96a75e.png)

Same for `deftype`

**Comment by danielcompton** [comment]
I'm not quite sure what the issue is here, but I suspect Cursive's resolution features won't work because of the decorate macro https://github.com/savagematt/bowen/blob/master/src/bowen/core.clj#L68.

**Comment by savagematt** [comment]
Yes- I also assume it's because the defrecord is nested. It seems like reasonable default behaviour to notice it anyway.

Although I could be wrong- I'm aware this may only be desirable behaviour in my specific circumstances.

**Comment by cursiveclojure** [comment]
@danielcompton is right, Cursive won't parse inside the `decorate` macro by default - I can fix that. 

@savagematt When you say it's nested, what do you mean? That it's nested inside the `decorate`?

**Comment by savagematt** [comment]
@cursiveclojure yes, that.

**Comment by savagematt** [comment]
Thanks Colin, as always.

**Comment by cursiveclojure** [comment]
No problem - I'll try to get that in soon, it's a one-liner.
